snap/snapcraft.yaml

```diff
@@ -325,6 +363,8 @@ parts:
       snapcraftctl build
       $SNAPCRAFT_STAGE/update-icon-cache.sh $SNAPCRAFT_PART_INSTALL/share/icons
       $SNAPCRAFT_STAGE/split-gtk-theme.sh $SNAPCRAFT_PART_INSTALL
+      # Copy Yaru to Yaru-light to support < Jammy
+      cp -rp $SNAPCRAFT_PART_INSTALL/share/themes/Yaru $SNAPCRAFT_PART_INSTALL/share/themes/Yaru-light
```

Comment:
Relative symlinks aren't working as well, right?

Reply:
Correct, snapd actually complains about the mount